README.md

```diff
@@ -182,6 +182,7 @@ The plan:
 
 `emergency.js` is not cached by browser and loaded before all.
 
-## Nested docs
+## Documentation
 
-1. [Amounts formatting](src/cow-react/utils/amountFormat/README.md)
+1. [Oveall Architecture](./docs//architecture-overview.md)
```

Comment:
Double slashes

```suggestion
1. [Oveall Architecture](./docs/architecture-overview.md)
```

Reply:
I saw it, and ammended the commit. but I never pushed the changes. fixed!